<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-88R09EB" failures="1" tests="4" name="Test" time="0.216" errors="0" timestamp="2023-09-10T22:25:28 IST">
  <testcase classname="com.testng.automaticretry.TestclassSample" name="passTest" time="0.031"/>
  <testcase classname="com.testng.automaticretry.TestclassSample" name="retryFailTest" time="0.008">
    <skipped/>
  </testcase> <!-- retryFailTest -->
  <testcase classname="com.testng.automaticretry.TestclassSample" name="retryFailTest" time="0.005">
    <skipped/>
  </testcase> <!-- retryFailTest -->
  <testcase classname="com.testng.automaticretry.TestclassSample" name="retryFailTest" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.testng.automaticretry.TestclassSample.retryFailTest(TestclassSample.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- retryFailTest -->
</testsuite> <!-- Test -->
